src/Model/Command/UserRecommendation.php

```diff
@@ -30,7 +32,9 @@ class UserRecommendation extends AbstractCommand implements UserAwareInterface
     /** @var string */
     private $minimalRelevance = self::MINIMAL_RELEVANCE_LOW;
     /** @var string[] */
-    private $filters = ['valid_to >= NOW'];
+    private $filters = [];
```

Comment:
Also phpdoc of `addFilter()` and `setFilters()` method needs to be updated to reflect this change (= there is no default filter).